_docs/data-model/v1/upload.md

```diff
@@ -55,6 +59,48 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+### deviceType
+This is the broad category of the device. In general, it should be one of the following:
+
+* `"pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"pump-cgm"` (for pumps integrated with a cgm)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
+* `"pen"` (for insulin injection devices)
+* `"manual"` (for manually entered data)
+
+For devices that do not fit in one of these categories, please contact Tidepool so we can standardize on a good name.
+
+### deviceManufacturer
+
+This is the name of the device manufacturer. We are trying to standardize this field as well, so please use exactly the following names:
+
+    Abbott
+    Asante
+    Dexcom
+    Insulet
+    Medtronic
+    OneTouch
```

Comment:
If we're going to standardize, then the equivalent to `OneTouch` would be `FreeStyle` instead of `Abbott`. Probably should be `LifeScan` instead to maintain the parallel (company name, not brand name?) There's a grey area with `Medtronic` vs. `MiniMed` there too.